app-shells/mksh/mksh-54.recipe

```diff
@@ -66,4 +65,4 @@ INSTALL()
 TEST()
 {
 	./test.sh
-}
+}
```

Comment:
This line is missing a linefeed. If you edit the file with vi and save it then vi will fix that for you.